pkg/operator/etcd_assets/bindata.go

```diff
@@ -343,7 +367,6 @@ MANIFEST_DIR="${CONFIG_FILE_DIR}/manifests"
 ETCD_DATA_DIR="/var/lib/etcd"
 ETCD_DATA_DIR_BACKUP="/var/lib/etcd-backup"
 MANIFEST_STOPPED_DIR="${ASSET_DIR}/manifests-stopped"
-RESTORE_ETCD_POD_YAML="${CONFIG_FILE_DIR}/static-pod-resources/etcd-certs/configmaps/restore-etcd-pod/pod.yaml"
```

Comment:
reminder we are keeping this in 4.5 right, we need to tolerate backups with and without

Reply:
Yeah.

Reply:
I was hoping that the automatic cherry-pick will fail, but would create the BZ for me for free :-)